src/App.jsx

```diff
@@ -5,8 +5,12 @@ import { Outlet } from 'react-router-dom';
 function App() {
   return (
     <>
-      <Navbar />
-      <Outlet />
+      <div className="navbar">
```

Comment:
Is there a reason to wrap the navbar with a div?. You could have that className inside the navbar and avoid extra tags.

Reply:
Sorry, I deleted this